tsl/test/sql/include/dist_query_run.sql

```diff
@@ -288,6 +288,100 @@ LIMIT 10;
 
 DROP TABLE join_test;
 
+-- Test SkipScan
+-- Ensure that a Unique plan gets chosen on the access node
+\set TEST_DESC '\n######### SELECT DISTINCT pushdown\n'
```

Comment:
Would be good to echo the comments instead of the same TEST_DESC strings for all the tests. Then it is possible to read see the information in the test output.